FreeRTOS/FreeRTOS
# Fix mpu simulator demo

Description
-----------
Past updates in MPU support have not been reflected in some of the MPU demos.  This PR updates the GCC project in the Demo/CORTEX_MPU_Simulator_Keil_GCC so it builds with the latest GCC, latest Keil simulator, and latest FreeRTOS code and MPU configuration options.

Test Steps
-----------
Build and run the demo in the simulator.



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
